AlphaWallet/Core/Helpers/RestartQueueHandler.swift

```diff
@@ -18,6 +18,19 @@ final class RestartQueueHandler {
         processRestartQueueBeforeRestart(config: config, restartQueue: restartQueue)
     }
 
+    func processRestartQueueAfterRestart(coordinator: InCoordinator, restartQueue: RestartTaskQueue) {
+        for each in restartQueue.queue {
+            switch each {
+            case .addServer, .reloadServers, .editServer, .removeServer, .enableServer, .switchDappServer:
+                break
+            case .loadUrlInDappBrowser(let url):
+                restartQueue.remove(each)
+                coordinator.showTab(.browser)
```

Comment:
But this makes `RestartQueueHandler` tightly coupled to `InCoordinator` right?

Reply:
just moved function, can extract `showTab ` to protocol to avoid using `InCoordonator`

Reply:
Yeah. Maybe. Anyway, good to go, just pointing out so we are wary of coupling across the app, not necessarily just here :) Can merge this first